agent/engine/docker_image_manager.go

```diff
@@ -303,21 +309,149 @@ func (imageManager *dockerImageManager) removeUnusedImages(ctx context.Context)
 	imageManager.updateLock.Lock()
 	defer imageManager.updateLock.Unlock()
 
-	imageManager.imageStatesConsideredForDeletion = imageManager.imagesConsiderForDeletion(imageManager.getAllImageStates())
+	var numECSImagesDeleted int
+	imageManager.imageStatesConsideredForDeletion = imageManager.imagesConsiderForDeletion(imageManager.getAllImageStates(), ctx)
```

Comment:
why ctx is added as an input? seems there is no where in **imagesConsiderForDeletion** using it.

Reply:
remove it.